crawl-ref/source/evoke.cc

```diff
@@ -235,8 +235,12 @@ static bool _evoke_horn_of_geryon()
         return false;
     }
 
+#if TAG_MAJOR_VERSION == 34
```

Comment:
Calling out this construction for review. I've used it several times in `evoke.cc`. You can see my comment in the commit body of cee19d09c69062020d22862ac1ad74ded89edd2c for why it's left here in TAG 35.

Reply:
Seems plausible. It's kinda a bummer to leave all these constant 0s lying around, but I imagine that ifdef'ing out all the places that result is used would be even less pleasant.

Reply:
Yeah, because the variable name changes (or at least it would be very misleading if it didn't change), you'd end up having to duplicate logic too, which I feel is just asking for someone to later come along and only update the logic in one place.

Reply:
That's definitely cleaner code (DRY at least), but given that once TAG is bumped the entire concept of surge will be removed, not sure it's worth my time. What do you think?